base/src/main/java/com/mindera/skeletoid/routing/OpenEmailCommand.kt

```diff
@@ -0,0 +1,31 @@
+package com.mindera.skeletoid.routing
+
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import com.mindera.skeletoid.R
+
+class OpenEmailCommand(
+    private val context: Context,
+    private val emailTo: Int,
+    private val emailSubject: Int,
+    private val emailBody: Int
+) : IRouteCommand {
+
+    override fun navigate() {
+        val mailData = context.let {
+            context.getString(
+                R.string.email_uri_data,
+                Uri.encode(context.getString(emailTo)),
+                Uri.encode(context.getString(emailSubject)),
+                Uri.encode(context.getString(emailBody))
+            )
+        }
+
+        val intent = Intent(Intent.ACTION_SENDTO).apply {
+            data = Uri.parse(mailData)
```

Comment:
I think it would be cleaner and easier to read if we did like this : https://stackoverflow.com/a/5333251

Reply:
Although that way is cleaner in terms of code, the chooser shows applications that are not only email applications (such as bluetooth, wifi direct, etc)

Reply:
Oh, that is crap. 
Did you try this one? https://stackoverflow.com/a/13849013

Reply:
I'm probably being picky. Never mind.